src/main/java/core/basesyntax/service/fileoperation/impl/CsvFileWriterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.fileoperation.impl;
+
+import core.basesyntax.service.fileoperation.CreateReport;
+import core.basesyntax.service.fileoperation.CsvFileWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterImpl implements CsvFileWriter {
+    private final CreateReport report = new CreateReportImpl();
+
+    @Override
+    public void writeFile(String filePath) {
```

Comment:
```suggestion
    public void writeFile(String filePath, String report) {
```

lets pass report as incoming parameter